plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -391,6 +392,22 @@ class ServicesTable extends React.Component {
     );
   }
 
+  renderInstances(prop, service) {
+    const instancesCount = service.getInstancesCount();
+    const tasksRunning = service.getTaskCount();
+    const overview = tasksRunning === instancesCount
+      ? ` ${tasksRunning}`
+      : ` ${tasksRunning}/${instancesCount}`;
+
+    const content = instancesCount == null ? "\u2014" : overview;
```

Comment:
ideally `instancesCount` will always have a number but to be safe and follow the design requirements we display `-`